<launch>

	<rosparam command="load" 
		file="$(find pointcloud_image_converter)/config/r3live_fp_pc2img.yaml" />

	<node pkg="pointcloud_image_converter" name="pc2img_node" type="pc2img_node" 
		output="screen" ns="pc2img" required="true">
		<remap from="input_cloud" to="/r3live/cloud_raw" />
	</node>

</launch >
